src/Uno.UI/UI/Xaml/Controls/TimePicker/TimePickerFlyout.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using Windows.UI.Xaml.Controls.Primitives;
+
+namespace Windows.UI.Xaml.Controls
+{
+	partial class TimePickerFlyout : PickerFlyoutBase
+	{
+#if !__IOS__
+		protected override Control CreatePresenter() => throw new NotImplementedException();
```

Comment:
these will be implemented eventually? Or they're simply not supported?

Reply:
TimePicker not supported on other platforms right now.  Will be udpated when we'll import TimePicker from WinUI.